src/Tribe/Views/V2/Customizer/Section/Global_Elements.php

```diff
@@ -0,0 +1,510 @@
+			],
+			'background_color'        => [
+				'sanitize_callback'	   => 'sanitize_hex_color',
+				'sanitize_js_callback' => 'maybe_hash_hex_color',
+			],
+			'accent_color'            => [
+				'sanitize_callback'	   => 'sanitize_hex_color',
+				'sanitize_js_callback' => 'maybe_hash_hex_color',
+			],
+		];
+	}
+
+	public function setup_content_headings() {
+		return [
+			'font_family_heading' => [
+				'priority'	 => 0,
+				'type'		 => 'heading',
+				'label'      => esc_html__( 'Select Font Family', 'the-events-calendar' ),
+			],
+			'font_size_heading' => [
+				'priority'	 => 5,
+				'type'		 => 'heading',
+				'label'      => esc_html__( 'Set Font Size', 'the-events-calendar' ),
+				'description' => esc_html_x(
+					'Choose a base font size. Event text will scale around the selected base size.',
+					'The description for the base font size setting.',
+					'the-events-calendar'
+				),
+			],
+			'font_color_heading' => [
+				'priority'	 => 10,
+				'type'		 => 'heading',
+				'label'      => esc_html__( 'Set Font Colors', 'the-events-calendar' ),
+			],
+			'global_elements_separator' => [
+				'priority'	 => 20,
+				'type'		 => 'separator',
+			],
+			'adjust_appearance_heading' => [
+				'priority'	 => 21,
+				'type'		 => 'heading',
+				'label'      => esc_html__( 'Adjust Appearance', 'the-events-calendar' ),
+			],
+		];
+	}
+
+	/**
+	 * {@inheritdoc}
+	 */
+	public function setup_content_controls() {
+		// Because Customizer doesn't show the default value.
+		if ( ! empty( $this->get_option( 'font_size_base' ) ) ) {
+			$font_size_base_value = $this->get_option( 'font_size_base' );
+		} else {
+			$font_size_base_value = $this->defaults[ 'font_size_base' ];
+		}
+
+		return [
+			'font_family'             => [
+				'priority' => 3,
+				'type'     => 'radio',
+				'choices' => [
+					'default'  => _x(
+						'Default',
+						'Label for option to use default TEC fonts.',
+						'the-events-calendar'
+					),
+					'theme'    => _x(
+						"Inherit theme font(s)",
+						'Label for option to use theme fonts.',
+						'the-events-calendar'
+					)
+				],
+			],
+			'font_size_base'          => [
+				'priority' => 6,
+				'type'     => 'number',
+				'label'    => esc_html_x(
+					'By Pixel',
+					'The base font size input setting label.',
+					'the-events-calendar'
+				),
+				'input_attrs' => [
+					'min'   => '8',
+					'max'   => '24',
+					'step'  => '1',
+					'style' => 'width: 4em;',
+					'value' => (int) $font_size_base_value,
+				]
+			],
+			'font_size'               => [
+				'priority' => 7,
+				'type'     => 'range-slider',
+				'label'    => esc_html_x(
+					'By Scale',
+					'The font size selector setting label.',
+					'the-events-calendar'
+				),
+				'input_attrs' => [
+					'min'  => -1,
+					'max'  => 1,
+					'step' => 1,
+					// Because there is no label for this input - we give screen readers something to work with.
+					'aria-described-by' => esc_html_x(
```

Comment:
`aria-describedby` needs IDs of the elements that describe this element. Rather, use `aria-description` or `aria-label`. I think `aria-label` is more suitable here, but you have a label above. I'm not sure what you mean by your comment here about there being no label for the input?

```suggestion
					'aria-label' => esc_attr_x(
```